finviz/helper_functions/request_functions.py

```diff
@@ -62,7 +63,7 @@ def sequential_data_scrape(
 ) -> List[Dict]:
     data = []
 
-    for url in tqdm(urls):
+    for url in tqdm(urls, disable=os.environ.get("DISABLE_TQDM", False):
```

Comment:
This doesn't actually work. Ex. `DISABLE_TQDM=False python example.py` will remove the progress bar. Environment variables are returned as `str` rather than bool.